modal/_serialization.py

```diff
@@ -28,10 +30,11 @@ def __init__(self, client, buf):
         super().__init__(buf)
 
     def persistent_load(self, pid):
-        object_id = pid
-        # TODO(erikbern): we should get the proto somehow,
-        # for functions
-        return _Handle._from_id(object_id, self.client, None)
+        (object_id, target_interface, handle_proto) = pid
+        raw_obj = _Handle._from_id(object_id, self.client, handle_proto)
+        if target_interface:
+            return async_utils.synchronizer._translate_out(raw_obj, target_interface)
```

Comment:
we talked about it on slack, but it would be nice to have just one interface and put the async stuff on top of the blocking stuff

Reply:
Yeah, that would be nice for these sort of things (I guess we'd still need to know if we are serializing a wrapper or not, but not the type of wrapper)